dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -2353,6 +2358,12 @@ public int getMyTid() {
   public static synchronized void HydraTask_startSnappyCluster() {
     if (forceStart) {
       startSnappyCluster();
+      try {
+        Thread.sleep(120000);
```

Comment:
Why sleep ? Can't we check for status file and fail after certain time.

Reply:
This is to avoid the stopSnappy cluster immediately after restart and verify the recovery in UI.